test/e2e/csv_e2e_test.go

```diff
@@ -4169,6 +4175,210 @@ var _ = Describe("ClusterServiceVersion", func() {
 
 		Expect(apiService2.Spec.CABundle).Should(Equal(apiService1.Spec.CABundle))
 	})
+
+	It("Disabling copied csvs", func() {
```

Comment:
nit: We may want to utilize something like a When/Context/Describe type container so we can utilize BeforeEach/AfterEach stanzas vs. nesting Context blocks.

Reply:
e2e has been updated to better align with Behavior-Driven Development  style.